pkg/opsgenie/it/client_manual_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	opsGenieIntManURL = "https://micros.prod.atl-paas.net"
+	opsgenieIntManURL = "https://micros.prod.atl-paas.net"
```

Comment:
Don't use real data in tests please. Use fake data =)

Reply:
This is a manual integration test, I don't think fake data is applicable.